### PR TITLE
Makes manifest preserve original position of Custom jobs

### PR DESCRIPTION
**Pull request by Heroman3003**
Title. Screw all modifications out of the general alt-titles throwing the position down to Miscellaneous.

### CODE REVIEW

File: code/datums/datacore.dm [hunk 1]
```diff
@@ -302,11 +302,18 @@ var/global/list/PDA_Manifest = list()
...
-			real_title = J.title
+		if(J.title == rank)					//If we have a rank, just default to using that.
+			real_title = rank
 			break
+		else if(J.title == assignment)
+			real_title = rank
+			break
+		else
+			var/list/alttitles = get_alternate_titles(J.title)
+			if(!J)	continue
```
Comment (MistakeNot4892) [inline code/datums/datacore.dm:313]: Why do you check `J` after the point where a falsey value in `J` would have runtimed?

Comment (Heroman3003) [reply code/datums/datacore.dm:313]: Thats the part I did not modify. I have not a single clue why it was like that in the first place, but it wasn't my place to judge.

Comment (MistakeNot4892) [reply code/datums/datacore.dm:313]: You can safely delete it. `J` is implicitly typechecked by the for loop.

Comment (Heroman3003) [reply code/datums/datacore.dm]: Sure. I was wandering why it was like that when its basically a check for an always false thing, but I decided that I'm a bit too sleepy to try and delete extremely ambigous-looking things that were there before me.

**Pull request merged**